modules/openapi-generator/src/main/resources/dart2/auth/oauth.mustache

```diff
@@ -3,7 +3,7 @@
 class OAuth implements Authentication {
   OAuth({this.accessToken});
 
-  String accessToken;
+  String? accessToken;
```

Comment:
by making this not-null you will avoid null check below. does it make sense to use OAuth without access token? Technically yes, but practically that's a sec vuln on endpoint?